fdb-extensions/src/test/java/com/apple/test/RandomizedTestUtils.java

```diff
@@ -47,12 +48,34 @@ private RandomizedTestUtils() {
     public static Stream<Arguments> randomArguments(Function<Random, Arguments> randomArguments) {
         if (includeRandomTests()) {
             Random random = ThreadLocalRandom.current();
-            return IntStream.range(0, getIterations()).mapToObj(i -> randomArguments.apply(random));
+            return Stream.generate(() -> randomArguments.apply(random))
+                    .limit(getIterations());
         } else {
             return Stream.of();
         }
     }
 
+    /**
+     * Return a stream of random {@code long}s to be used to seed random number generators.
+     * This can be supplied to a {@link org.junit.jupiter.params.ParameterizedTest} to run a test
+     * under different scenarios. This is preferred over, say, a {@code Stream<Random>} because
+     * the seed can be included in the test display name, which means that a failing seed can be
+     * recorded and the test re-run with that seed.
+     *
+     * @param staticSeeds a set of seeds to always include in the returned random seeds
+     * @return a stream of random {@code long}s to initialize {@link Random}s
+     */
+    @Nonnull
+    public static Stream<Long> randomSeeds(long... staticSeeds) {
```

Comment:
Given the test here, perhaps it would make sense for this to include one global seed if none are provided, or always. That seems like it would make it easier to use, and harder to accidentally have your test not run with any seeds.